packages/chakra-ui/src/AlertDialog/test.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from "react";
+import {
+  render,
+  fireEvent,
+  waitForElement,
+  waitForElementToBeRemoved,
+  prettyDOM,
+} from "../utils/testing";
+import AlertDialog from "./index";
+import {
+  AlertDialogHeader,
+  AlertDialogBody,
+  AlertDialogFooter,
+} from "./components";
+
+const MockDialog: React.FunctionComponent = () => {
```

Comment:
I really like the idea of mocking the components in the test but it might make some of the files very long. Do you think there's enough value in moving mocked components to a mock directory?

Reply:
I find that co-locating the tests and mock components is helpful and keeps the tests pretty simple. If the files are getting long, I think it is best to split up isolates tests into their own files rather than separate the mocks/tests.